tests/tests.cpp

```diff
@@ -42,33 +44,331 @@ Graph createGraph1() {
+    REQUIRE(graph->findShortestPath(graph->getNode("56"), graph->getNode("24"), 1) == 1814.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("56"), graph->getNode("43"), 1) == 660.0); 
+    REQUIRE(graph->findShortestPath(graph->getNode("56"), graph->getNode("7"), 1) == 1861.0);
+
+    REQUIRE(graph->findShortestPath(graph->getNode("8"), graph->getNode("3"), 1) == 684.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("8"), graph->getNode("32"), 1) == 1040.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("8"), graph->getNode("56"), 1) == 1396.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("8"), graph->getNode("8"), 1) == 0.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("8"), graph->getNode("24"), 1) == 1256.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("8"), graph->getNode("43"), 1) == 945.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("8"), graph->getNode("7"), 1) == 693.0);
+
+    REQUIRE(graph->findShortestPath(graph->getNode("7"), graph->getNode("3"), 1) == 1378.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("7"), graph->getNode("32"), 1) == 1734.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("7"), graph->getNode("56"), 1) == 1859.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("7"), graph->getNode("8"), 1) == 694.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("7"), graph->getNode("24"), 1) == 1345.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("7"), graph->getNode("43"), 1) == 1200.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("7"), graph->getNode("7"), 1) == 0.0);
+
+    REQUIRE(graph->findShortestPath(graph->getNode("43"), graph->getNode("3"), 1) == 1120.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("43"), graph->getNode("32"), 1) == 763.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("43"), graph->getNode("56"), 1) == 659.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("43"), graph->getNode("8"), 1) == 946.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("43"), graph->getNode("24"), 1) == 2202.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("43"), graph->getNode("43"), 1) == 0.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("43"), graph->getNode("7"), 1) == 1201);
+
+    REQUIRE(graph->findShortestPath(graph->getNode("24"), graph->getNode("3"), 1) == 1101.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("24"), graph->getNode("32"), 1) == 1457.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("24"), graph->getNode("56"), 1) == 1813.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("24"), graph->getNode("8"), 1) == 1255.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("24"), graph->getNode("24"), 1) == 0.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("24"), graph->getNode("43"), 1) == 2200.0);
+    REQUIRE(graph->findShortestPath(graph->getNode("24"), graph->getNode("7"), 1) == 1344.0);
+
+    // // miles (passes)
```

Comment:
if you guys have this commented, why not get rid of it? is there a reason to keep it?

Reply:
no we'll keep it just because calculating all the things by hand took a lot of time and don't want to just make it go to waste and plus i think it might be of help in the future